bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/protocol/RemoteControllerWebSocket.java

```diff
@@ -149,7 +149,7 @@ public void openConnection() throws RemoteControllerException {
                 // websocket connect will be done in lifetime handler
                 return;
             } catch (Exception e) {
```

Comment:
Is it also possible to change the catch to RuntimeException (+ any thrown checked exception) instead of Exception?

Reply:
see above